.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
   build_old:
     strategy:
       matrix:
-        go-version: [1.16.x, 1.17.x]
```

Comment:
Updating the golang version is done in the go.mod. This just changes which go versions we build against.

Reply:
ah I see, then maybe change this PR name to "Update CI to build against 1.17 and 1.18"